src/content/api/batch-types/farmlands-external-asset.mdx

```diff
@@ -85,6 +85,10 @@ Exported Farmlands Credit Card information used for importing and updating of a
     Unique 9 digit field to display in barcode format to pay by [Patron Code](/api/patron-codes).
   </Property>
 
+  <Property name="number" type="bignumber">
```

Comment:
I think potentially "primary account number" or PAN might be a more specific way of naming this field. Could you ask Neza or a contact at Farmlands if this would be correct? https://en.wikipedia.org/wiki/Payment_card_number